code/plot_gii_surface.py

```diff
@@ -0,0 +1,38 @@
+""" 
+
+Example usage:
+
+python plot_gii_surface.py /path/to/my/mesh /path/to/my/map.gii \
+                          <hemisphere> /path/to/my/img.png \
+                          [view]
+
+"""
+
+import nilearn
+import sys
+from nilearn import plotting
+
+def main():
+
+  
+  in_mesh = sys.argv[1]    # Surface mesh (.gii or Freesurfer specific files such as .orig, 
```

Comment:
Can you use an argumentparser, such as from `argparse` instead? It's much more robust and can provide help-text to users. The current solution would just crash without an informative error message if all the arguments weren't provided. You can look at any of the other scripts I uploaded for an example.